### PR TITLE
integration: auth dropin

### PR DESCRIPTION
**Pull request by sirugh**
#26 was merged accidentally, and after several revert commits, we have to create the branch anew.

This is approved, and just requires a QA pass.

### Test URLs:
https://auth-dropin-again--boilerplate-commerce-dropins--hlxsites.hlx.live/customer/login
https://auth-dropin-again--boilerplate-commerce-dropins--hlxsites.hlx.live/customer/create
https://auth-dropin-again--boilerplate-commerce-dropins--hlxsites.hlx.live/customer/forgotpassword
https://auth-dropin-again--boilerplate-commerce-dropins--hlxsites.hlx.live/customer/account/createPassword/?email=test%40test.com&id=224&token=1WxpuC

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
       "@dropins/storefront-order-confirmation/": "/scripts/__dropins__/storefront-order-confirmation/",
       "@dropins/storefront-auth/": "/scripts/__dropins__/storefront-auth/"
     }
+  }
```
Comment (sirugh) [inline head.html:22]: oops!

Comment (Ferri0) [reply head.html]: I'm assuming that 4 space indentation is correct based on previous code block in head.html?

Screenshot 2024-05-30 at 20 02 31

Comment (sirugh) [reply head.html]: Not sure it matters here, I was mostly commenting that we had missed the closing bracket :) Not sure how it was working.

**Comment by dpatil-magento** [comment]
QA observations - 
1. Auth User > Order confirmation page does not load.
2. Being on forgot password page, if user logs in using sign-in underlay then user is still remains on forgot password page and does NOT redirected to my account or home page.
3. Reset password URL has user email in it. Not big deal but I have not seen this in any other websites while password reset.  (Dont remember, but i provided this feedback some where, please ignore if this is already discussed).
4. Combined Auth Modal overlay does not close on hitting Esc key.
5. Even after login, Combined Auth header button still there. 
6. Cart, Checkout, PDP does load well in Mobile device but Combined Auth Modal overalys, SignIn Underlay, My account Underlay, SignIn text, My account page after login does not look good in mobile device.

**Comment by Ferri0** [comment]
Hi @dpatil-magento,

I've reviewed your observations. Please take a look at the following details:

1. Auth User > Order confirmation page does not load.
_This will be covered in the scope of checkout drop-in, no changes from auth drop-in side required. The integration with checkout will be delivered as a separate PR, when required functionality and slots will be implemented on Checkout side. No action points here from our side._
2. Being on forgot password page, if user logs in using sign-in underlay then user is still remains on forgot password page and does NOT redirected to my account or home page.
_We will fix that ✅_ 
3. Reset password URL has user email in it. Not big deal but I have not seen this in any other websites while password reset. (Dont remember, but i provided this feedback some where, please ignore if this is already discussed).
_It is expected behaviour, functionality implemented in https://jira.corp.adobe.com/browse/LYNX-397. No action points here._
4. Combined Auth Modal overlay does not close on hitting Esc key.
_We will fix that ✅_ 
5. Even after login, Combined Auth header button still there.
_We will fix that ✅_ 
6. Cart, Checkout, PDP does load well in Mobile device but Combined Auth Modal overalys, SignIn Underlay, My account Underlay, SignIn text, My account page after login does not look good in mobile device.
_We will double check and adjust UI ✅_

**Comment by dpatil-magento** [comment]
Created follow up ticket https://jira.corp.adobe.com/browse/USF-1281

**Pull request merged**